scala-unit-tests/src/test/scala/org/eclipse/collections/impl/list/mutable/SynchronizedMutableListScalaTest.scala

```diff
@@ -28,6 +28,15 @@ class SynchronizedMutableListScalaTest extends SynchronizedMutableCollectionTest
         }
     }
 
+    @Test
+    def sort_synchronized
+    {
+        this.assertSynchronized
+        {
+            this.classUnderTest.sort(null)
+        }
+    }
+
```

Comment:
Is sortThis tested in the same class?

Reply:
Good point.  Added tests for both forms of sortThis.